Jenkinsfile

```diff
@@ -14,39 +14,40 @@ pipeline {
 	  }
     }
     stage('Deploy') {
-      when {
-         not { changeRequest() }
-      }
```

Comment:
So if I understand well, we want to deploy also PRs builds ? I think this is good but it will require to setup a cleanup task (manual or automatic) for the nightly download area.

Reply:
Yes, times to times we need to test the whole product.